modules/kafka/testcontainers/kafka/__init__.py

```diff
@@ -1,12 +1,15 @@
+import base64
 import tarfile
 import time
+import uuid
 from io import BytesIO
 from textwrap import dedent
 
 from testcontainers.core.container import DockerContainer
 from testcontainers.core.utils import raise_for_deprecated_parameter
+from testcontainers.core.version import ComparableVersion
 from testcontainers.core.waiting_utils import wait_for_logs
-from testcontainers.kafka._redpanda import RedpandaContainer
```

Comment:
?

Reply:
sorry about that, it was my IDE auto-removing unused imports, already fixed it 👍